### DISCUSSION TITLE
Connectivity matrix issues with Jumper and Line

### DISCUSSION THREAD

**Discussion by PandaRP01** [CURENT/andes#473] [Q&A] [answered]
Dear Dr. Cui,
While simulating a electrical network, I observed an issue which is very strange.

There is a `Jumper` between two buses and `Jumper.u=0`.

**Case-1:**
There is a Jumper between two buses and `Jumper.u=0.`

`system.PFlow.config.check_conn=1` and run the `PFlow`, 
In this case, nether PFlow shows error nor stops. 
If I forcefully stops, then in the error massage, it is showing, checking connectivity.

**Case-2:**
There is a Jumper between two buses and `Jumper.u=0.`

`system.PFlow.config.check_conn=0` and run the `PFlow`, 
In this case, PFlow converg and TDS is also running fine

**Case-3:**
There is a Jumper between two buses and `Jumper.u=0`. 
Also I put a line in between that two buses, `Line.r=0`, and` Line.x=1e-8` and `Line.u=0`

`system.PFlow.config.check_conn=1` and run the `PFlow,` 
In this case, PFlow converg and TDS is also running fine

Could you please suggest, why this behavior of the system and how to resolve it.

Also could you suggest why **Case-3** is running which is very much similar to **Case-1**.

Regards
Rakesh Panda, Ph.D.

**Comment by cuihantao** [comment]
It looks like the connectivity check might not be handling Jumper correctly. Allow me some time to come up with a fix.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: PandaRP01 ***@***.***>
Sent: Wednesday, August 9, 2023 10:01:28 AM
To: CURENT/andes ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [CURENT/andes] Connectivity matrix issues with Jumper and Line (Discussion #473)


Dear Dr. Cui,
While simulating a electrical network, I observed an issue which is very strange.

There is a Jumper between two buses and Jumper.u=0.

Case-1:
There is a Jumper between two buses and Jumper.u=0.

system.PFlow.config.check_conn=1 and run the PFlow,
In this case, nether PFlow shows error nor stops.
If I forcefully stops, then in the error massage, it is showing, checking connectivity.

Case-2:
There is a Jumper between two buses and Jumper.u=0.

system.PFlow.config.check_conn=0 and run the PFlow,
In this case, PFlow converg and TDS is also running fine

Case-3:
There is a Jumper between two buses and Jumper.u=0.
Also I put a line in between that two buses, Line.r=0, and Line.x=1e-8 and Line.u=0

system.PFlow.config.check_conn=1 and run the PFlow,
In this case, PFlow converg and TDS is also running fine

Could you please suggest, why this behavior of the system and how to resolve it.

Also could you suggest why Case-3 is running which is very much similar to Case-1.

Regards
Rakesh Panda, Ph.D.

**Reply by PandaRP01** [reply]
Thank you so much Dr. Cui for attending my issues. I am eagerly waiting for your response.

Regards
Rakesh

**Reply by cuihantao** [reply, marked as answer]
It has been fixed on the `develop` branch. A new release 1.8.10 is on the way.

**Reply by PandaRP01** [reply]
Thank you Dr. Cui
